de.nicokimmel.shadowcast-electron.json

```diff
@@ -0,0 +1,51 @@
+{
+  "app-id": "de.nicokimmel.shadowcast-electron",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "23.08",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.node18"
+  ],
+  "base": "org.electronjs.Electron2.BaseApp",
+  "base-version": "23.08",
+  "command": "run.sh",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=x11",
+    "--device=all",
+    "--socket=pulseaudio"
+  ],
+  "build-options": {
+    "append-path": "/usr/lib/sdk/node18/bin",
+    "env": {
+      "NPM_CONFIG_LOGLEVEL": "info"
+    }
+  },
+  "modules": [
+    {
+      "name": "shadowcast-electron",
+      "buildsystem": "simple",
+      "build-commands": [
+          "npm install",
+          "npm run build",
+          "cp -r dist/linux-unpacked/* /app/build",
+          "install -Dm755 run.sh /app/bin/run.sh"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/nicokimmel/shadowcast-electron.git",
+          "tag": "v1.0.1",
+          "commit": "54a545450ee902bfbe0a4f295a285feb4960dc79"
+        },
+        {
+          "type": "script",
+          "commands": [
+            "exec /usr/bin/env electron /app/build/shadowcast-electron"
```

Comment:
Pretty sure you'd need to use zypak for the electron sandbox to work in flatpak.

Reply:
> Pretty sure you'd need to use zypak for the electron sandbox to work in flatpak.

Done